src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -217,9 +225,19 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
             this.resetTabCount();
 
             if (!breakdownLabel) {
-              navigateToDrilldownPage(getDrilldownSlug(), this);
+              const slug = this.getPageSlug();
+              if (slug) {
+                navigateToDrilldownPage(slug, this);
+              } else {
+                throw new Error(`Invalid page slug ${slug}`);
+              }
             } else {
-              navigateToValueBreakdown(getDrilldownValueSlug(), breakdownLabel, this);
+              const valueSlug = this.getDrilldownValueSlug();
+              if (valueSlug) {
+                navigateToValueBreakdown(valueSlug, breakdownLabel, this);
+              } else {
+                throw new Error(`Invalid value slug ${valueSlug}`);
+              }
```

Comment:
I wonder if we could gracefully handle these situations, in case the user mistakenly lands here.